### ISSUE TITLE
`import --help` usage example doesn’t mention file arg

### ISSUE THREAD

**Issue by brandonkelly** [craftcms/nitro#124]
The Usage example that’s output when you run `nitro import --help` doesn’t mention that there’s a required file argument:

```sh
$ nitro import --help
Import database into a machine

Usage:
  nitro import [flags]

Flags:
  -h, --help   help for import

Global Flags:
  -d, --debug            Show command output and do not execute.
  -m, --machine string   Name of a machine.
```

But it’s clearly required:

```sh

$ nitro import
Error: requires at least 1 arg(s), only received 0
```

**Comment by jasonmccallister** [comment]
We can add `examples` to each command and it would output like so:

```
$ nitro import -h
Import database into a machine

Usage:
  nitro import [flags]

Examples:
import my-backup.sql

Flags:
  -h, --help   help for import

Global Flags:
  -d, --debug            Show command output and do not execute.
  -m, --machine string   Name of a machine.
```

We should definitely update the usage on the command, but we would need to alter quite a bit of the commands to create a custom template. There is also a 1.0.0 release to the package we are using that has some nice upgrades. I'm going to look into that effort and see if they have made it easier to define usage.

**Comment by jasonmccallister** [comment]
This will be released in 1.0.0-beta.3!
